core/templates/css/oppia-material.css

```diff
@@ -379,6 +379,14 @@ input.mat-input-element {
   padding-top: 8px;
   padding-bottom: 8px; }
 
+.mat-radio-group-questions .mat-radio-button .mat-radio-label {
+  align-items: flex-start;
+}
+
+.mat-radio-group-questions .mat-radio-button .mat-radio-label .mat-list-item {
+  height: auto;
+}
+
```

Comment:
Editing this file is not allowed. Please try to find a different place for these files.

Reply:
> Editing this file is not allowed. Please try to find a different place for these changes.
Ok, Just saw the file description that it can't be modified, will add it in oppia.css with proper comment why it is required. Thanks !

Reply:
Moved the changes to oppia.css. Please take a look @srijanreddy98. Thanks!